javascript/app-framework/src/components/app/DataShaperApp/DataShaperApp.types.ts

```diff
@@ -39,6 +39,11 @@ export interface DataShaperAppProps {
 	 */
 	defaultHelp?: string
 
+	/**
+	 * Object of user-definable application-level settings.
+	 */
+	defaultSettings?: any
```

Comment:
Can we create a type alias for this to avoid having intentional `any's sprinkled too far. 

Maybe `type DataShaperSettings = any`